### PR TITLE
Enhancement/check node version for fire editor

### PR DESCRIPTION
**Pull request by HristiyanG**

### CODE REVIEW

File: doc/developer-documentation/aeproject-cli/fire-editor.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 The integration between the aeproject and the [Fire Editor aepp](http://fireeditor.nikitafuchs.de/) allows the user to compile and deploy contracts, call contract's function from different accounts using the **Fire Editor aepp**. 
 
+### Prerequisite
+
+Please note that the *fire-editor* is running on top of Anular CLI 8.0+
```
Comment (maptuhec) [inline doc/developer-documentation/aeproject-cli/fire-editor.md]: missing G in Angular

File: doc/developer-documentation/aeproject-cli/fire-editor.md [hunk 2]
```diff
@@ -4,6 +4,11 @@
...
+The official Node.js version of *Anular CLI 8.0+* that is supported is **10.9.0** or greater
```
Comment (maptuhec) [inline doc/developer-documentation/aeproject-cli/fire-editor.md]: also here

File: packages/aeproject-cli/aeproject-fire-editor/fire-editor.js [hunk 3]
```diff
@@ -29,6 +29,11 @@ const moduleName = constants.MODULE_NAME;
 const isWindowsPlatform = process.platform === 'win32';
 
 const run = async (options) => {
+    let result = await isNodeVersionSupported();
+    
+    if (!(result >= 0)) {
```
Comment (maptuhec) [inline packages/aeproject-cli/aeproject-fire-editor/fire-editor.js]: result < 0 ?

File: packages/aeproject-cli/aeproject-fire-editor/fire-editor.js [hunk 4]
```diff
@@ -155,6 +159,51 @@ const installModuleDependencies = async () => {
...
+    if (!fireEditorTokens.every(isValidPart) || !localNodeTokens.every(isValidPart)) {
+        return NaN;
+    }
+
+    for (var i = 0; i < fireEditorTokens.length; ++i) {
+
+        if (fireEditorTokens[i] == localNodeTokens[i]) {
+            continue;
+        } else if (localNodeTokens[i] > fireEditorTokens[i]) {
+            return 1;
```
Comment (maptuhec) [inline packages/aeproject-cli/aeproject-fire-editor/fire-editor.js]: why returning 1,0,-1 but not true and false?

File: packages/aeproject-cli/aeproject-fire-editor/fire-editor.js [hunk 5]
```diff
@@ -155,6 +159,51 @@ const installModuleDependencies = async () => {
...
+        } else {
+            return -1;
+        }
+    }
+
+    if (fireEditorTokens.length != localNodeTokens.length) {
```
Comment (maptuhec) [inline packages/aeproject-cli/aeproject-fire-editor/fire-editor.js]: what whill hapen if local version is 10.13?

File: packages/aeproject-cli/aeproject-fire-editor/fire-editor.js (hunk 3 above)
Comment (HristiyanG) [reply packages/aeproject-cli/aeproject-fire-editor/fire-editor.js]: 1 - greater
0 - equal
-1 lower
if i am checking the positive scenario all the try-catch should be enclosed inside the if

File: packages/aeproject-cli/aeproject-fire-editor/fire-editor.js (hunk 5 above)
Comment (HristiyanG) [reply packages/aeproject-cli/aeproject-fire-editor/fire-editor.js]: fixed it

**Pull request merged**